AggregateMetrics/AggregateMetrics/AzureWebApp/Implementation/CacheHelper.cs

```diff
@@ -68,22 +76,22 @@ public int PerformanceCounterValue(string performanceCounterName, string json)
         /// <returns>value from cache</returns>
         public int GetCounterValue(string name)
         {
-            const string jsonKey = "json";
-            if (!CacheHelper.Instance.IsInCache(jsonKey))
+            const string JonKey = "json";
```

Comment:
Correct the spelling of "Json" (currently it says "Jon" which is a person's name).